boutdata/cbdtoeqdsk.py

```diff
@@ -1,8 +1,9 @@
 from __future__ import print_function
 from boututils.file_import import file_import
-from bunch import Bunch
 import numpy as np
 
+class Bunch:
+    pass
```

Comment:
One could consider using the same pattern as shown in the [`boututils` PR](https://github.com/boutproject/boututils/blob/pylib_merge/boututils/bunch.py) until `boutdata` is cleaned

Reply:
This doesn't make a difference as Bunch is not initialised with a dict - thus it is exactly the same.
It might still break somewhere because other features of Bunch are expected - but I couldn't find any use.